Content.Server/StationEvents/Events/MassHallucinationsRule.cs

```diff
@@ -2,39 +2,48 @@
 using Content.Server.StationEvents.Components;
 using Content.Server.Traits.Assorted;
 using Content.Shared.GameTicking.Components;
+using Content.Shared.Humanoid;
 using Content.Shared.Mind.Components;
 using Content.Shared.Traits.Assorted;
+using Robust.Shared.Collections;
 
 namespace Content.Server.StationEvents.Events;
 
 public sealed class MassHallucinationsRule : StationEventSystem<MassHallucinationsRuleComponent>
 {
     [Dependency] private readonly ParacusiaSystem _paracusia = default!;
 
+    private ValueList<EntityUid> _toChange = new();
+
```

Comment:
This needs to be stored on the component itself as I mentioned or else it will break with save-load.